modules/ROOT/pages/io-connectors.adoc

```diff
@@ -0,0 +1,55 @@
+= Luna Streaming IO Connectors
+
+When you have Luna Streaming installed and running, add IO connectors to your deployment to interact with external systems, such as Apache Cassandra, Aerospike, and many others.
```

Comment:
Aerospike is not included in Luna Streaming. See https://github.com/datastax/release-notes/blob/master/Luna_Streaming_2.8_Release_Notes.md#builtin-connectors